modules/distr-tracing-rn-new-features.adoc

```diff
@@ -0,0 +1,38 @@
+////
+Module included in the following assemblies:
+* distr-tracing--release-notes.adoc
+////
+////
+Feature – Describe the new functionality available to the customer. For enhancements, try to describe as specifically as possible where the customer will see changes.
+Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.). For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.
```

Comment:
Typically we only use parenthesis for acronyms. 

What do you think about this rewrite: 
`Reason – If known, include why the enhancement has been implemented, like for the use case, performance, or technology. For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.`

Reply:
This is a comment.  It's not published.  =)